### PR TITLE
Replace nbody to njoint and activate all collision pairs by default

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

File: src/multibody/model.hxx [hunk 1]
```diff
@@ -240,35 +240,35 @@ namespace se3
   inline Data::Data (const Model & ref)
     :model(ref)
```
Comment (jmirabel) [inline src/multibody/model.hxx]: This has to be double checked as I am not sure of where those values are being used.

File: src/multibody/model.hxx [hunk 2]
```diff
@@ -31,7 +31,7 @@ namespace se3
   inline std::ostream& operator<< (std::ostream & os, const Model & model)
   {
     os << "Nb bodies = " << model.nbody << " (nq="<< model.nq<<",nv="<<model.nv<<")" << std::endl;
-    for(Model::Index i=0;i<(Model::Index)(model.nbody);++i)
```
Comment (jmirabel) [inline src/multibody/model.hxx]: I guess I should change `"Nb bodies  = " << model.nbody` by  `"Nb joints  = " << model.njoint` ?

Comment (jcarpent) [inline src/multibody/model.hxx]: Yes, you can operate as you mentioned

**Comment by jmirabel** [comment]
Solve #259

File: src/multibody/model.hxx [hunk 3]
```diff
@@ -238,8 +238,7 @@ namespace se3
 
 
   inline Data::Data (const Model & ref)
```
Comment (jcarpent) [inline src/multibody/model.hxx]: I think you can replace ref by model, to be more precise

**Comment by jcarpent** [comment]
I think a simple rebase on the current level branch should help travis.

**Comment by jcarpent** [comment]
Seems good to me. I will merge.

**Pull request merged**